CMakeLists.txt

```diff
@@ -15,9 +15,7 @@ set(COTIO_LICENSE "Modified Apache 2.0 License")
 
 project(copentimelineio VERSION ${COTIO_VERSION} LANGUAGES CXX C)
 
-if (POLICY CMP0077)
```

Comment:
this bit here is because we are using a cmake minimum of 3.9. Can you also change the cmake version universally to 3.13, the minimum cmake that understands the new behavior of the 77 policy? I don't recall a reason that we must use cmake 3.17 minimum on Windows, unless there is something broken versus recent versions of visual studio, so I think simply setting the min to 3.13 makes the most sense.

Reply:
I believe the old cmake rev was to accommodate some very old linux distros, but I don't think that's important at all for the C bindings.

Reply:
Ok, I'll change it to 3.13 :+1: